lake/pool.go

```diff
@@ -239,11 +242,22 @@ func (p *Pool) StoreInStaging(ctx context.Context, txn *commit.Transaction) erro
 	return storage.Put(ctx, p.engine, p.StagingObject(txn.ID), bytes.NewReader(b))
 }
 
-func (p *Pool) ScanStaging(ctx context.Context, w zio.Writer, ids []ksuid.KSUID) error {
+// ScanStaging writes the provided commits in staging to the provided
+// zio.Writer.  If not commit tags are passed, all commits in staging are
+// written.
```

Comment:
Nit: I think it's clearer to refer to parameters directly by name rather than indirectly by type or description. (The standard library largely follows this style.)
```suggestion
// ScanStaging writes the staging commits in ids to w.
// If ids is empty, all staging commits are written.
```

Reply:
This is much simpler! Thanks!